src/librustc_privacy/lib.rs

```diff
@@ -194,58 +197,103 @@ impl<'a, 'tcx> EmbargoVisitor<'a, 'tcx> {
     fn exported_trait(&self, _id: ast::NodeId) -> bool {
         true
     }
+
+    fn is_public_exported_ty(&self, ty: &hir::Ty) -> (bool, bool) {
```

Comment:
Could you add a comment to this and the function below explaining the tuple return value? Just saying which bool is which is fine